docs/user/managing-devices.md

```diff
@@ -423,16 +423,26 @@ You can deploy, update, or undeploy applications on a device by updating the lis
 
 The following table shows the application runtimes and formats supported by Flight Control:
 
-| Runtime | Descriptor Format | Package Format | Package Repository | Note |
-| ------- | ----------------- | -------------- | ------------------ | ---- |
-| Podman | [podman-compose](https://github.com/containers/podman-compose) | OCI image with compose spec | OCI registry | Requires `podman-compose` to be installed on the device. |
+### Runtime: **Podman**
+
+| Specification      | Format                        | Source / Delivery                  |
+| ------------------ | ----------------------------- | ---------------------------------- |
+| Compose specification (via [`podman-compose`](https://github.com/containers/podman-compose)) | OCI image                          | OCI registry                      |
+| Compose specification (via [`podman-compose`](https://github.com/containers/podman-compose)) | Unpackaged (inline or embedded)    | Embedded or inline in project     |
+
+> [!NOTE]
+> Requires `podman-compose` to be installed on the device.
+
+> [!TIP]
+> Short image names (e.g., `nginx`) are not supported. Use fully qualified references like `docker.io/nginx` to avoid ambiguity.
 
 To deploy an application to a device, create a new entry in the "applications" section of the device's specification, specifying the following parameters:
 
 | Parameter | Description |
 | --------- | ----------- |
-| Name | A user-defined name for the application. This will be used when the web UI and CLI list applications. |
-| Image | A reference to an application package in an OCI registry. |
+| Name    | A user-defined name for the application. This will be used when the web UI and CLI list applications. |
+| Image   | A reference to an application package in an OCI registry. |
+| AppType | The application format type. |
```

Comment:
Do we mention somewhere which values this can take? We talk about `podman-compose` above, but here expect the type to be just `compose`, or?